datafusion/core/tests/sqllogictests/test_files/array.slt

```diff
@@ -189,34 +190,67 @@ select make_array(NULL), make_array(NULL, NULL, NULL), make_array(make_array(NUL
 ----
 [] [] [[], []]
 
-# make_array with columns #1
-query ????
-select make_array(a), make_array(b, c), make_array(d), make_array(e) from values;
-----
-[1] [1, 2] [1.1] [Lorem]
-[2] [3, 4] [2.2] [ipsum]
-[3] [5, 6] [3.3] [dolor]
-[4] [7, 8] [4.4] [sit]
-[0] [9, 10] [5.5] [amet]
-[5] [0, 12] [6.6] [,]
-[6] [11, 0] [7.7] [consectetur]
-[7] [13, 14] [0.0] [adipiscing]
-[8] [15, 16] [8.8] []
-
-# make_array with columns #2
+# make_array with 1 columns
+query ???
+select make_array(a), make_array(d), make_array(e) from values;
```

Comment:
I broke out the tests for `make_array` with a single column from the tests of `make_array` with 2 column